indico/htdocs/js/indico/jquery/declarative.js

```diff
@@ -303,4 +304,29 @@
             }).html('&para;').appendTo($elem);
         });
     }
+
+    function setupToggleLinks() {
+        $('[data-toggle]:not([data-toggle=dropdown])').on('click', function(evt) {
```

Comment:
Wait, this is not truly declarative! It runs once and binds to the elements that match the selector at that point, not ones added in the future! It actually stops working after editing e.g. a comment. You want this instead:

    $('body').on('click', '[data-toggle]:not([data-toggle=dropdown])', function(evt) {